src/app/static/src/tests/components/regionPage.test.ts

```diff
@@ -44,7 +44,8 @@ describe("region page", () => {
     it("sets current step when step is clicked", async () => {
 
         const store = createStore();
-        const wrapper = mount(regionPage, {localVue, store, router});
+        const stubs = ["interventions"];
```

Comment:
Unfortunately dynamic imports break the tests. Stubbing seems to be the workaround, though I guess you're now no longer testing that the component import works at all.